cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -17,13 +25,16 @@
 
 @JacocoGenerated
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-@JsonPropertyOrder({"@context"})
+@JsonPropertyOrder({CONTEXT, ID, TYPE_PROPERTY, IDENTIFIERS, NAMES, CONTACT_DETAILS, IMAGE, AFFILIATIONS})
```

Comment:
Here we are having a strange coupling with the constant values and the field names of the class. I am not sure why we need the property ordering, but  if we do we should the following:
```
@JsonProperty(ID)
    private URI id;
    @JsonProperty(FIRST_NAME)
    private String firstName;
    @JsonProperty(LAST_NAME)
    private String lastName;

    @JsonCreator
    public Person(@JsonProperty(ID) URI id,
                  @JsonProperty(FIRST_NAME) String firstName,
                  @JsonProperty(LAST_NAME) String lastName) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
    }
```